src/db/MariadbIfc.py

```diff
@@ -400,14 +399,14 @@ def getProfile(self,
 
         return profile
 
-    def getProfiles(self,
+    def getUsersProfiles(self,
```

Comment:
nit: functions should be listed in alphabetical order, except for 'main' (or its equivalent) at the bottom.